docs/Earn/liquid-staking/bnb/bnb-liquid-staking-mechanics.md

```diff
@@ -5,7 +5,7 @@ id: bnb-liquid-staking-mechanics
 
 # Staking Mechanics
 
-ANKR Liquid Staking lets the user stake their funds through the corresponding smart contracts on ANKR, accumulate rewards for it, and receive their stake and rewards when unstaking.
+BNB Liquid Staking lets the user stake their funds through the corresponding smart contracts on the Binance network, accumulate rewards for it, and receive their stake and rewards when unstaking.
 
 It is well-described by its requirement, fees, rewards, validators, as well as smart contracts and workflow that shows the functions to call to interact with these smart contracts.
```

Comment:
The following section explains Staking requirements, fees, rewards, validators, smart contracts and function calls to interact with these smart contracts.

Reply:
done